backend/tests/server.cpp

```diff
@@ -37,13 +37,14 @@ TEST_CASE("Test endpoints using handler")
     crypto_core_ristretto255_scalar_random(b);
 
     // 2. encrypt each password with b (and hash to point)
-    std::vector<std::string> encrypted_passwords = cryptography::encrypt(passwords, b);
+    const size_t offset = 1;
+    std::vector<std::string> encrypted_passwords = cryptography::encrypt(passwords, b, offset);
 
     // 3. insert into database
     for (const auto &password : encrypted_passwords)
     {
         // encode password before inserting into database
-        db.execute("INSERT INTO passwords (password) VALUES ('" + crow::utility::base64encode(password, password.size()) + "');");
+        db.execute("INSERT INTO passwords (password) VALUES ('" + crow::utility::base64encode(password, password.size() + offset) + "');");
```

Comment:
**issue**: base64encode() should not include the offset in the size, since the offset is included in the password.size()